optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/constraintweight/descriptor/ConstraintConfigurationDescriptor.java

```diff
@@ -118,7 +118,8 @@ private void processPackAnnotation(DescriptorPolicy descriptorPolicy) {
         // So the ConstraintWeightDescriptors parse packAnnotation.constraintPackage() themselves.
         constraintPackage = packAnnotation.constraintPackage();
         if (constraintPackage.isEmpty()) {
-            constraintPackage = constraintConfigurationClass.getPackage().getName();
+            Package pack = constraintConfigurationClass.getPackage();
```

Comment:
I wanted to call this variable "package", but that's a reserved keyword in Java.